### PR TITLE
Mixed OKLCH color with white

### PR DESCRIPTION
**Pull request by usmanyunusov**
### Before:
before

### After:
after

### CODE REVIEW

**Comment by ai** [comment]
Looks good!

Can you rebase you changes? (Import `blend` directly from `culori/fn` since we added types there)

File: view/base/index.css [hunk 1]
```diff
@@ -8,8 +8,8 @@
 
 :root {
   /* Colors */
-  --surface-0: oklch(96% 0.005 300);
-  --surface-1: oklch(100% 0 0);
+  --surface-0: oklch(96% 0.005 300 / 100%);
```
Comment (ai) [inline view/base/index.css:11]: Can you revert it? `100%` is default value

Comment (usmanyunusov) [reply view/base/index.css]: Revered!

File: view/chart/paint.ts [hunk 2]
```diff
@@ -32,10 +32,10 @@ function paintDot(
   alpha: number
 ): void {
   if (alpha < 1) {
-    ctx.fillStyle = background
-    ctx.fillRect(x, y, width, height)
     color.alpha = alpha
```
Comment (ai) [inline view/chart/paint.ts:35]: Do we need to use `color.alpha` if we want to avoid using alpha?

Comment (usmanyunusov) [reply view/chart/paint.ts:35]: Without `color.alpha` `color` has `alfa` value 1

Comment (ai) [reply view/chart/paint.ts:35]: This is the point to color with alpha 1.

Comment (ai) [reply view/chart/paint.ts:35]: We can blend `blend([color, bg, bg])` for Rec2020 and `blend([color, color, bg])` for P3 to have different value without using alpha.

Comment (ai) [reply view/chart/paint.ts:35]: Do we have any problems with riding `alpha` completely?

Comment (usmanyunusov) [reply view/chart/paint.ts]: Снимок экрана 2022-05-22 в 12 43 01

https://github.com/evilmartians/oklch-picker/pull/38/files#diff-d4b214cdd799ad8e568b91d6dcb24d1dd94541bbbf7f4db9f671863bcc9b45eeR31

For the chart `paint.ts` removed `alpha`.

Comment (usmanyunusov) [reply view/chart/paint.ts]: https://culorijs.org/api/#blend

The `blend` only mixes alpha channels normally

Comment (ai) [reply view/chart/paint.ts]: > The blend only mixes alpha channels normally

I didn’t find it in docs. Example in docs uses `alpha` just and example.

Quote from the docs:

> A separable blend mode is a simple formula that gets applied to each channel in the color space independently

File: view/chart/paint.ts [hunk 3]
```diff
@@ -28,14 +26,20 @@ function paintDot(
...
-  if (alpha < 1) {
-    ctx.fillStyle = background
-    ctx.fillRect(x, y, width, height)
-    color.alpha = alpha
+  let bgColor = parseRgb(background)
+
+  if (!inRGB(color) && inP3(color)) {
+    color.alpha = 0.6
+    bgColor.alpha = 1 - color.alpha
+    color = blend([format(color), format(bgColor)], 'screen')
```
Comment (ai) [inline view/chart/paint.ts:36]: You do not need to call `format()` here. `blend()` works with `Color` object from ` culori`.

**Comment by ai** [comment]
Culori’s `blend()` mixes only in RGB mode, which make everything meanless. We will not maker a good result here.

**Pull request closed without merging**